orders/views.py

```diff
@@ -1,13 +1,21 @@
 import json
-from unicodedata import name
 
-import jwt
-
-from django.http  import JsonResponse
-from django.views import View
+from django.http     import JsonResponse
+from django.views    import View
 
 from orders.models         import Cart
-from core.token_dacorators import token_decorator
+from core.token_decorators import token_decorator
+
+class CartView(View):
+    @token_decorator
+    def delete(self, requset, cart_id):
+        try:
+            Cart.objects.get(id = cart_id, user = requset.user).delete()
+
+            return JsonResponse({'message' : 'No Content'}, status = 200)
+        
+        except Cart.DoesNotExist:
+            return JsonResponse({'message' : 'Invalid Cart'}, status = 400)
```

Comment:
데이터가 존재 하지 않을 때 상태코든 주로 404를 사용합니다.
상태 코드에 대해서 조금 더 학습하면 좋을 것 같습니다.

```suggestion
        except Cart.DoesNotExist:
            return JsonResponse({'message' : 'Invalid Cart'}, status = 404)
```